### DISCUSSION TITLE
Edit base.yml

### DISCUSSION THREAD

**Discussion by aagha** [espanso/espanso#1919] [Feature requests and Ideas]
I'd love to see "Edit base.yml" or "Edit Espanso config" as an option in the menu for Espanso.

**Comment by AucaCoyan** [comment]
Thanks! I would love too! Added to the features list

**Comment by smeech** [comment]
Menu, what menu? :laughing:

**Reply by aagha** [reply]
![image](https://github.com/espanso/espanso/assets/5522103/22ced050-efe0-4890-9522-123545c4bcb9)

PS - The icon shows up as white on Windows.

**Reply by smeech** [reply]
Sorry - trying to be humorous - Linux doesn't even have the icon. :cry:

**Reply by AucaCoyan** [reply]
Oh, I didn't remember that linux hasn't a tray icon. That is a good feature to implement too.
_edit: gnome has no sense of a tray icon so to speak_
https://www.reddit.com/r/rust/comments/dz1t5a/how_to_add_tray_icons/f84pn06/
